### PR TITLE
Cloudwatch: Update variable query docs

### PR DESCRIPTION
**Pull request by iwysiu**
**What this PR does / why we need it**:
Removes references to the old template string format since the CloudWatch Variable Editor became a form in v8.5.0:
Screen Shot 2022-04-26 at 5 56 42 PM

**Which issue(s) this PR fixes**:

Part of #48007 

**Special notes for your reviewer**:
This review removes more docs than it adds, as presumably the form itself will provide guidance.

### CODE REVIEW

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 1]
```diff
@@ -13,51 +13,36 @@ For an introduction to templating and template variables, refer to the [Templati
...
-| `dimension_values(us-east-1,AWS/Redshift,CPUUtilization,ClusterIdentifier)`                                                   | RedShift         |
-| `dimension_values(us-east-1,AWS/RDS,CPUUtilization,DBInstanceIdentifier)`                                                     | RDS              |
-| `dimension_values(us-east-1,AWS/S3,BucketSizeBytes,BucketName)`                                                               | S3               |
-| `dimension_values(us-east-1,CWAgent,disk_used_percent,device,{"InstanceId":"$instance_id"})`                                  | CloudWatch Agent |
-| `resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})` | ELB              |
-| `resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"Component":["$service"],"Environment":["$environment"]})`        | ELB              |
-| `resource_arns(eu-west-1,ec2:instance,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})`                      | EC2              |
+Refer to the [variable syntax documentation]({{< relref "../../variables/syntax.md" >}}).
```
Comment (andresmgot) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:36]: maybe you can add a note saying that using variables in the variable form is supported?

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 2]
```diff
@@ -13,51 +13,36 @@ For an introduction to templating and template variables, refer to the [Templati
...
 
 ## Using JSON format template variables
 
 Some queries accept filters in JSON format and Grafana supports the conversion of template variables to JSON.
 
-If `env = 'production', 'staging'`, following query will return ARNs of EC2 instances which `Environment` tag is `production` or `staging`.
+If `env = 'production', 'staging'`, a `Resource ARNs` query with the following filter will return ARNs of EC2 instances where the `Environment` tag is equal to `production` or `staging`.
```
Comment (andresmgot) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:42]: I am missing a "For example, if ...` at the beginning here.

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 3]
```diff
@@ -13,51 +13,36 @@ For an introduction to templating and template variables, refer to the [Templati
...
-| `dimension_values(us-east-1,AWS/Redshift,CPUUtilization,ClusterIdentifier)`                                                   | RedShift         |
-| `dimension_values(us-east-1,AWS/RDS,CPUUtilization,DBInstanceIdentifier)`                                                     | RDS              |
-| `dimension_values(us-east-1,AWS/S3,BucketSizeBytes,BucketName)`                                                               | S3               |
-| `dimension_values(us-east-1,CWAgent,disk_used_percent,device,{"InstanceId":"$instance_id"})`                                  | CloudWatch Agent |
-| `resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})` | ELB              |
-| `resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"Component":["$service"],"Environment":["$environment"]})`        | ELB              |
-| `resource_arns(eu-west-1,ec2:instance,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})`                      | EC2              |
+Refer to the [variable syntax documentation]({{< relref "../../variables/syntax.md" >}}).
 
 ## Using JSON format template variables
```
Comment (andresmgot) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:38]: maybe you can backport https://github.com/grafana/grafana/pull/47907 to 8.5 so all the changes are together and you don't need JSON format anymore (or is that not an option for some reason?)

**Review by andresmgot — APPROVED** [review]
Thanks for the docs! some suggestions here

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md (hunk 2 above)
Comment (iwysiu) [reply docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md]: done

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 4]
```diff
@@ -13,51 +13,36 @@ For an introduction to templating and template variables, refer to the [Templati
 
 ## Query variable
 
-The CloudWatch data source provides the following queries that you can specify in the `Query` field in the Variable edit view. They allow you to fill a variable's options list with things like `region`, `namespaces`, `metric names` and `dimension keys/values`.
-
-In place of `region` you can specify `default` to use the default region configured in the data source for the query,
-e.g. `metrics(AWS/DynamoDB, default)` or `dimension_values(default, ..., ..., ...)`.
+The CloudWatch data source provides the following queries that you can specify in the `Query Type` field in the Variable edit view. They allow you to fill a variable's options list with things like `region`, `namespaces`, `metric names` and `dimension keys/values`.
```
Comment (chri2547) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:16]:
```suggestion
The CloudWatch data source provides the following queries that you can specify in the `Query Type` field in the Variable edit view. They enable you to fill a variable's options list with values such as `region`, `namespaces`, `metric names` and `dimension keys/values`.
```

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 5]
```diff
@@ -13,51 +13,36 @@ For an introduction to templating and template variables, refer to the [Templati
...
-| `resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"Component":["$service"],"Environment":["$environment"]})`        | ELB              |
-| `resource_arns(eu-west-1,ec2:instance,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})`                      | EC2              |
+Variables can be used in the variable form. Refer to the [variable syntax documentation]({{< relref "../../variables/syntax.md" >}}).
 
 ## Using JSON format template variables
 
 Some queries accept filters in JSON format and Grafana supports the conversion of template variables to JSON.
 
-If `env = 'production', 'staging'`, following query will return ARNs of EC2 instances which `Environment` tag is `production` or `staging`.
+For example, if `env = 'production', 'staging'`, a `Resource ARNs` query with the following filter will return ARNs of EC2 instances where the `Environment` tag is equal to `production` or `staging`.
```
Comment (chri2547) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:42]:
```suggestion
For example, if `env = 'production', 'staging'`, a `Resource ARNs` query with the following filter returns ARNs of EC2 instances where the `Environment` tag is equal to `production` or `staging`.
```

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 6]
```diff
@@ -113,10 +92,4 @@ Only 1 attribute per instance can be returned. Any flat attribute can be selecte
...
 - `VpcId`
 
-Tags can be selected by prepending the tag name with `Tags.`
-
-Example `ec2_instance_attribute()` query
-
-```javascript
-ec2_instance_attribute(us - east - 1, Tags.Name, { 'tag:Team': ['sysops'] });
-```
+Tags can be selected by prepending the tag name with `Tags.`. For example, the tag `Name` would be selected with `Tags.Name`.
```
Comment (chri2547) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:95]:
```suggestion
You can select tags by prepending the tag name with `Tags.`. For example, the tag `Name` is selected with `Tags.Name`.
```

**Review by chri2547 — CHANGES_REQUESTED** [review]
A few suggestions.

**Review by chri2547 — APPROVED** [review]
LGTM!

**Review by sunker — APPROVED** [review]
Nice!

**Pull request merged**